math/mathcore/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ set(MATHCORE_HEADERS TRandom.h
 if(veccore)
   set(MATHCORE_LIBRARIES ${VecCore_LIBRARIES})
   set(MATHCORE_BUILTINS VECCORE)
+  set(MATHCORE_HEADERS ${MATHCORE_HEADERS} TMathVectorized.h)
```

Comment:
Please do not make this conditional. Just disable the code inside the header itself if it's not supposed to be enabled.